Update doc to reflect current terms (#4842)
jenkins-infra/jenkins.io: content/projects/gsoc/students.adoc

---
layout: project
title: "Google Summer of Code. Information for GSoC Contributors"

> description: "This page aggregates links to internal and external resources for Google Summer of Code Contributors"

Reviewer: Since contributors is not a title, I think it should be lower case.
```suggestion
description: "This page aggregates links to internal and external resources for Google Summer of Code contributors"
```

=== Application steps

. Check out the link:/projects/gsoc/2022/project-ideas[GSoC 2022 Project ideas]
. Select an interesting project idea or draft your own proposal.
. Use the link:https://docs.google.com/document/d/1dIlPLXfLbFsvcaHFuwmH9_lSCVm9m6-SgNYTNAnSZpY/[project proposal template] to write your project proposal.
. If you are not familiar with Jenkins, read the introductory info on the website and try using Jenkins with one of your previous projects.

> . Join the link:https://community.jenkins.io[Discourse communication channel]  

Reviewer: Show the GSoC topics rather than just showing the top level page?
```suggestion
. Join the link:https://community.jenkins.io/c/contributing/gsoc/[Discourse communication channel]
```

. A section related to testing your project: unit tests, integration tests, test automation.
. A section or paragraph on deliverables: demos, presentations, releases (alpha, official).
. A section or paragraph on improvements, bug fixes, benefits to the community.

> . Any other section that the contributor deem is applicable and helps the proposal

Reviewer: Singular contributor?
```suggestion
. Any other section that the contributor deems is applicable and helps the proposal
```

especially in the case of another jobs or internship.
. Unexpected events: we understand there can be unexpected events in life, and those cannot be planned. Please inform us as soon as possible if you need time away from the program. You can use private messaging for sensitive information.

> NOTE: Please note that the link:https://community.jenkins.io[Discourse channel]

Reviewer: ```suggestion
NOTE: Please note that the link:https://community.jenkins.io/c/contributing/gsoc/[Discourse channel]
```

is publicly visible inside and outside the community.
It is required to use this channel for the initial review and feedback collection.

> === First begin a communication in Discourse via https://community.jenkins.io

Reviewer: ```suggestion
=== First begin a communication in link:https://community.jenkins.io/c/contributing/gsoc/[Discourse]
```

== Coding periods

GSoC contributors are expected to...

. Work on the GSoC project as it is a full-time employment.

>  * It means 12 weeks of writing good code, about 30..40 hours per week is an **expected** workload though it can be adjusted upon the agreements with mentors.

Reviewer: I thought that they only expected half time work, not full time work during the twelve weeks.
```suggestion
 * It means 12 weeks of writing good code, about 20..30 hours per week is an **expected** workload though it can be adjusted upon the agreements with mentors.
```
